### PR TITLE
Issue 26 mission geometry

### PR DESCRIPTION
**Pull request by petrus7**
count mission gemetry
change representation of mission and asset geometry from wkt string to json objects

### CODE REVIEW

**Comment by tdipisa** [comment]
@petrus7 the travis build is failing

File: api/service/models.py [hunk 1]
```diff
@@ -55,7 +55,11 @@ class Meta:
     modified = DateTimeField(auto_now=True)
     description = TextField(blank=True, null=True)
     note = TextField(blank=True, null=True)
-    geometry = GeometryField(blank=True, null=True, srid=settings.DEFAULT_SRID)
+    geometry = LineStringField(blank=True, null=True, srid=settings.DEFAULT_SRID)
```
Comment (MV88) [inline api/service/models.py]: Why this?
Asset can also be a Point

File: api/service/models.py [hunk 2]
```diff
@@ -71,10 +75,11 @@ class Meta:
     description = TextField(blank=True, null=True)
     note = TextField(blank=True, null=True)
     created = DateTimeField(auto_now_add=True)
-    geometry = GeometryField(blank=True, null=True, srid=settings.DEFAULT_SRID)
+    geometry = LineStringField(blank=True, null=True, srid=settings.DEFAULT_SRID)
```
Comment (tdipisa) [inline api/service/models.py]: Why you have changed it from Geometry to LineString? Using Geometry is better.

File: api/service/models.py (hunk 1 above)
Comment (petrus7) [reply api/service/models.py]: Fixed

File: api/service/models.py (hunk 2 above)
Comment (petrus7) [reply api/service/models.py]: Restored

Comment (tdipisa) [reply api/service/models.py]: @petrus7, I still see LinesString

**Pull request merged**